corehq/apps/hqwebapp/templatetags/hq_shared_tags.py

```diff
@@ -39,11 +38,9 @@ def JSON(obj):
     try:
         return escape_script_tags(json.dumps(obj, default=json_handler))
     except TypeError as e:
-        msg = ("Unserializable data was sent to the `|JSON` template tag.  "
-               "If DEBUG is off, Django will silently swallow this error.  "
+        msg = ("Unserializable data was sent to a template tag that expectes JSON. "
                "{}".format(str(e)))
-        soft_assert(notify_admins=True)(False, msg)
```

Comment:
Removed the soft assert entirely because I've never heard of anyone looking at these, much less using them to fix an issue.